opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -72,27 +72,40 @@ def unwrap(obj, attr: str):
         setattr(obj, attr, func.__wrapped__)
 
 
-def get_token_context_span_kind(env, getter):
-    """Based on presence of active span, extracts context and initializes token and span_kind
+def start_internal_or_server_span(
+    tracer, span_name, start_time, env, context_getter
```

Comment:
`env` should be called `context`

Reply:
changed to `context_carrier` as `context` is already imported module and context is extracted from `env` object.
Let me know if you want me to rename it.